src/Sylius/Bundle/CoreBundle/Form/Type/LocaleType.php

```diff
@@ -46,6 +46,7 @@ public function __construct($dataClass, array $validationGroups = [], Repository
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        parent::buildForm($builder, $options);
```

Comment:
It would be quite more readable if there were an empty line after this statement.